lib/webenginesettings.cpp

```diff
@@ -110,17 +127,32 @@ void SailfishOS::WebEngineSettings::initialize()
     engineSettings->setPreference(QStringLiteral("intl.accept_languages"),
                                   QVariant::fromValue<QString>(langs));
 
+    // Ensure the renderer is configured correctly
+    engineSettings->setPreference(QStringLiteral("gfx.webrender.force-disabled"),
+                                  QVariant(true));
+    engineSettings->setPreference(QStringLiteral("embedlite.compositor.external_gl_context"),
+                                  QVariant(false));
```

Comment:
I think embedding.js (embedlite omni.ja) should default to false and browser set it true.

Reply:
I've removed these settings here and have them default to `false`, with some code in the browser initialisation to set the to `true`.